terraform/dynamodbwarehouse.tf

```diff
@@ -16,33 +15,26 @@ resource "aws_dynamodb_table" "basic-dynamodb-table" {
     type = "S"
   }
 
-  attribute {
-    name = "Amount_of_goods"
-    type = "N"
-  }
-
-  attribute {
-    name = "Price_of_goods"
-    type = "N"
-  }
-
-  global_secondary_index {
-    name               = "IndexForAmountOfGoods"
-    hash_key           = "Goods_Name"
-    range_key          = "Amount_of_goods"
-    write_capacity     = 10
-    read_capacity      = 10
-    projection_type    = "INCLUDE"
-    non_key_attributes = ["description"]
-  }
+global_secondary_index {
+    name = "IndexForGoodsName"
+    hash_key = "Goods_Name"
+    write_capacity = 10
+    read_capacity = 10
+    projection_type = "INCLUDE"
+    non_key_attributes = ["Amount_of_goods", "price_of_goods"]
+    }
+}
 
-  global_secondary_index {
-    name               = "IndexForPriceOfGoods"
-    hash_key           = "Goods_Name"
-    range_key          = "Price_of_goods"
-    write_capacity     = 10
-    read_capacity      = 10
-    projection_type    = "INCLUDE"
-    non_key_attributes = ["description"]
+# Example Item to check the Table
+resource "aws_dynamodb_table_item" "warehouse_item_example" {
```

Comment:
do you still need this?

Reply:
Yes that will be delete later for the first time i need this